javascript/commons/BattleRoyale.js

```diff
@@ -478,6 +487,9 @@ liquipedia.battleRoyale = {
 
 			this.attachHandlers( instanceId );
 			this.makeCollapsibles( instanceId );
+			if ( this.isMobile() ) {
```

Comment:
feels like this should be done with css tbh

Reply:
Do you mean by using content? https://developer.mozilla.org/en-US/docs/Web/CSS/content

Reply:
I was more thinking having both versions in the html and using media queries to show one or the other. That feels more natural to me

Reply:
I removed the changes to js and added a second header value to the Lua code. Together with bootstraps media query classes it shows/hides the text